docs/bits/keyword/letting_domain.md

```diff
@@ -21,7 +21,7 @@ Types of domains:
 - [set](...)
 - [multi-set](...)
 - [function](https://github.com/conjure-cp/conjure/blob/main/docs/bits/type/function.md)
```

Comment:
I am not sure about the file naming any more.. We should take a pass on this, starting with debugging the vs-code extension in S2. I'll merge these for now. Ideally we want the `function` token to fetch this documentation, and I suspect that's not happening when the filename isn't L_function.

Reply:
the extension picks up the function doc I believe...